packages/components/src/data-store/hooks/use-user-vote.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export default function useUserVote(props) {
+
+  let self = this;
+
+  const projectId = props.projectId || props.config.projectId;
+  const type = props.type || props.config.type;
```

Comment:
Ik zie vaker dat er ids op twee manieren opgehaald kunnen worden. Is dit niet iets wat we willen corrigeren? Bijvoorbeeld een projectId zit altijd op het hoogste niveau in props en niet in config? Of moeten variabelen ook in de config te definieren zijn? Naar mijn idee is dat onnodige verwarring

Reply:
Ik denk dat we dat willen fixen inderdaad, in ieder geval dat je het moet checken op twee niveaus. In mijn ogen hangt het af van hoe we config en props precies willen doorgeven, en dat weten we als we Rudi's service en jouw nieuwe opzet in elkaar schuiven